sessionctx/variable/sysvar.go

```diff
@@ -269,7 +270,6 @@ var defaultSysVars = []*SysVar{
 	{ScopeNone, "performance_schema_max_file_classes", "50"},
 	{ScopeGlobal, "expire_logs_days", "0"},
 	{ScopeGlobal | ScopeSession, "binlog_rows_query_log_events", "OFF"},
-	{ScopeGlobal, "validate_password_policy", "1"},
```

Comment:
why do we remove this?

Reply:
According to [sql reference](https://dev.mysql.com/doc/refman/8.0/en/validate-password-options-variables.html#sysvar_validate_password_policy), `This validate_password plugin system variable is deprecated and will be removed in a future version of MySQL.` 
So I presume we will not need it in the future. What do you think?

Reply:
@alapha23 I think we can discuss whether to remove these unused variables in another PR. For this PR, let's focused on the validation of `validate_password_number_count` and `validate_password_length`.